tests/feedback.test.ts

```diff
@@ -22,30 +34,63 @@ afterAll(async () => {
 
 describe('feedback submission', () => {
   test('properly persists on successful submission', async () => {
+    const event = EventFactory.fake({
```

Comment:
Can you see if there's any examples/any ways to have this as a global event, since it's created a couple different times in this test suite?

Reply:
Addressed with a function :)